integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/script/AddHeadersUsingNashornJsTestCase.java

```diff
@@ -73,7 +75,8 @@ public void testAddHeaderUsingOmElement() throws Exception {
         String payload = "<m:CheckPriceResponse xmlns:m=\"http://services.samples/xsd\">\n"
                 + "<m:stringHead>STRING</m:stringHead>\n" + "    <m:domHead>DOM</m:domHead>\n"
                 + "<m:omHead>OM</m:omHead>\n" + "</m:CheckPriceResponse>";
-        HttpResponse response = doPost(new URL(getApiInvocationURL("addHeadersWithNashornJs")), payload, httpHeaders);
+        HttpResponse response = doPost(new URL(getApiInvocationURL("addHeadersWithNashornJsAPI")), payload,
+                httpHeaders);
```

Comment:
Can't we have the the api name as a global variable so that we do not need to repeat it?